src/transformers/models/llama/tokenization_llama.py

```diff
@@ -31,7 +31,8 @@
 
 
 if TYPE_CHECKING:
-    from ...pipelines.conversational import Conversation
+    from transformers.pipelines.conversational import Conversation
```

Comment:
Out of curiosity, why is this an absolute import but the one below is not? I seem to recall we prefer relative imports over absolute ones in `transformers`

Reply:
Switched back to relative imports in all cases! I'm not sure how that one happened.